typescript-selenium-webdriver/tests/index.test.ts

```diff
@@ -7,6 +7,9 @@ import * as path from 'path';
 import { Builder, By, ThenableWebDriver, until } from 'selenium-webdriver';
 import * as chrome from 'selenium-webdriver/chrome';
 import { promisify } from 'util';
+import * as Axe from 'axe-core';
+
+const TEST_TIMEOUT_MS = 30000;

```

Comment:
Same comment here to move these constants to a different file which can be common! I wont block a PR on this though.

Reply:
I would normally agree, but thought maybe keeping everything visible in context was more valuable in this particular case for sample purposes. Will consider moving this + some of the ChromeDriver setup into a separate setup file in a separate PR